src/common/types/vector_constants.cpp

```diff
@@ -9,8 +9,8 @@ namespace duckdb {
 // In the case of the FlatVector/ConstantVector, they point towards static regions of memory
 // Hence in this case these cause no problems, as the destructors are non-trivial but effectively nops
 #ifdef __clang__
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wexit-time-destructors"
+//#pragma clang diagnostic push
+//#pragma clang diagnostic ignored "-Wexit-time-destructors"
```

Comment:
I think these are required in the OSX builds currently, see [here](https://github.com/duckdb/duckdb/blob/master/src/CMakeLists.txt#L12).

Perhaps we should just remove the warning entirely, or explicitly disable the warning for these files using the CMake instead.

Reply:
Thanks. One file is ours, the other is vendored. What's the process of updating the vendored version (if we wanted to erase the check there)?

Reply:
We added the check there, so update in any way you want